scripts/build_addon.sh

```diff
@@ -0,0 +1,64 @@
+# !/bin/bash
+
+if [ "$#" -eq 0 ]; then
+	echo "usage build_addon <product path> <release/debug - optional>"
+	exit
```

Comment:
```
if [ "$#" -lt 1 ]; then
```
makes more sense. Also, as a convention, optional args are marked with "[]", like so:
```
echo "usage build_addon <product path> [release/debug]"
```
Instead of 'build_addon', using $1.
Lastly, should always exit with a non zero RC in the event of an error.